grafana-plugin/src/containers/UserSettings/parts/tabs/PhoneVerification/PhoneVerification.tsx

```diff
@@ -39,7 +39,7 @@ const PhoneVerification = observer((props: PhoneVerificationProps) => {
   const { userStore, teamStore } = store;
 
   const userPk = (propsUserPk || userStore.currentUserPk) as User['pk'];
-  let user = userStore.items[userPk];
```

Comment:
old approach:
```tsx
let user = userStore.items[userPk];

// later down in the file
user = userStore.items[userPk];
```

after refactor:
```tsx
const [user, setUser] = useState<User>(userStore.items[userPk]);

// later down in the file
setUser(userStore.items[userPk]);
```
I believe `useState` is the proper approach here